### PR TITLE
build: Update to header 1.3.262

### PR DESCRIPTION
**Pull request by mikes-lunarg**
- Update known-good
- Generate source

### CODE REVIEW

File: include/vulkan/utility/vul_dispatch_table.h [hunk 1]
```diff
@@ -62,31 +63,31 @@ typedef struct VulInstanceDispatchTable_ {
     PFN_vkCreateDisplayPlaneSurfaceKHR CreateDisplayPlaneSurfaceKHR;
 #ifdef VK_USE_PLATFORM_XLIB_KHR
     PFN_vkCreateXlibSurfaceKHR CreateXlibSurfaceKHR;
-#endif  // VK_USE_PLATFORM_XLIB_KHR
+#endif //VK_USE_PLATFORM_XLIB_KHR
```
Comment (charles-lunarg) [inline include/vulkan/utility/vul_dispatch_table.h:66]: Did the file accidentally get run with clang-format before being checked in?

Seems like a reason to fix the code-gen, but for now a big change here is fine.

Comment (juan-lunarg) [reply include/vulkan/utility/vul_dispatch_table.h:66]: I'd rather avoid the diff in this PR

Comment (mikes-lunarg) [reply include/vulkan/utility/vul_dispatch_table.h:66]: Yes it looks like the following commit had clang-format applied before checking in: https://github.com/KhronosGroup/Vulkan-Utility-Libraries/commit/2bd0e0d43884f1d655d66c2195539834a113f850

I didn't run clang-format on this PR so all those formatting changes got reverted

**Review by juan-lunarg — APPROVED** [review]

File: include/vulkan/utility/vul_dispatch_table.h (hunk 1 above)
Comment (charles-lunarg) [reply include/vulkan/utility/vul_dispatch_table.h:66]: I'm going ahead and working on a fix so that its clang-format compliant.

**Pull request merged**